### PR TITLE
New indicator for weapons required for Damascus

### LINKED ISSUE #2: [Suggestion] Title bar of non-required weapons.
On phone, I have the 'hide non required' unchecked so I can work on them all (sometimes I'll feel like using one particular weapon for a match). 

However, there's no easy way to see which weapons actually work towards Damascus without having that checked, so I'm scrolling up and down to check it on/off.

I'd like to suggest to tint the non required weapons title bar color a different shade.

### PR DESCRIPTION
**Pull request by matt-whiteley**
Remove required circle. 
Added left border.
Adjusted scss appropriately. 
Fixes #2 

This is my first time working with vue, so it may not be perfect but it worked on my machine!

### CODE REVIEW

**Comment by matt-whiteley** [comment]
I have noticed an issue with the text on the combat knife weapon due to the border.

File: src/components/Weapons.vue [hunk 1]
```diff
@@ -4,9 +4,8 @@
       <h2>{{ title }}s</h2>
       <transition-group name="fade" tag="div" class="weapons">
         <div class="weapon" v-for="weapon in category" :key="weapon.alias">
-          <span class="required-circle" v-if="weapon.required"></span>
           <div class="name" 
-               :class="{ completed: Object.values(weapon.progress).every(Boolean) }" 
+               :class="{ completed: Object.values(weapon.progress).every(Boolean), 'required-weapon': weapon.required }"
```
Comment (carlssonemil) [inline src/components/Weapons.vue:8]:
```suggestion
               :class="{ completed: Object.values(weapon.progress).every(Boolean), 'required': weapon.required }" 
```
I'd change the classname to just 'required', to keep it simple.

File: src/components/Weapons.vue [hunk 2]
```diff
@@ -155,6 +134,11 @@ export default {
             background: $yellow;
             color: black;
           }
+
+          &.required-weapon {
```
Comment (carlssonemil) [inline src/components/Weapons.vue:138]:
```suggestion
          &.required {
```

File: src/components/Weapons.vue [hunk 3]
```diff
@@ -155,6 +134,11 @@ export default {
...
+            border-left: 5px solid $purple;
+            padding: 25px 25px 25px 20px; // to maintain text center alignment with the extra thick border
```
Comment (carlssonemil) [inline src/components/Weapons.vue:140]:
```suggestion
            padding: 25px 25px 25px 20px; // to maintain text center alignment with the extra thick border
            
            @media (max-width: $tablet) {
              padding: 15% 15% 15% calc(15% - 5px) // to maintain text center alignment with the extra thick border
            }
```
Don't forget the padding change on mobile/tablet.

**Pull request merged**